### DISCUSSION TITLE
Simulate "Follow" click button

### DISCUSSION THREAD

**Discussion by ShoshanaTzi** [tfsaggregator/tfsaggregator#367] [Q&A]
Hi,
Is it possible to create a rule that simulates a click on the "Follow" button for a specific work-item?
![image](https://user-images.githubusercontent.com/24288859/124437550-db042880-dd7f-11eb-995e-0df864956162.png)

**Comment by jessehouwing** [comment]
I don't know. I'm not aware of how the follow feature works under the hood. If it updates a field on the workitem, then this is likely possible, if if does anything else, then you're probably out of luck.

Though, it may be able to add support for this feature if it's currently not possible. I'd accept a pull request.

**Reply by ShoshanaTzi** [reply]
Ok, Thanks
